app/templates/power-ups/topics.html

```diff
@@ -183,6 +183,7 @@ <h2 id="data-storage">Data Storage</h2>
   <pre>t.set('board', 'private', 'key', value)</pre>
   <p>or</p>
   <pre>t.set('board', 'private', { key1: value1, key2: value2 })</pre>
+  <p>Storing at the 'organization' scope will only work if the active user is a member of the team to which the board belongs.</p>
```

Comment:
Good call. Updated and added a bit about recommended fall-back for when organization fails.